cli/config/configuration-nemu.toml.in

```diff
@@ -187,7 +187,7 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # If true and vsocks are supported, use vsocks to communicate directly
 # with the agent (no proxy is started).
 # Default true
-use_vsock = true
+#use_vsock = true
```

Comment:
Does this have to be commented out?

Reply:
this matches how it is handled in QEMU

Reply:
aah. well given that default is true in itself it doesn't matter.